Tella/Components/ServerConnectionHeaderView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ServerConnectionHeaderView.swift
+//  Tella
+//
+//  Created by gus valbuena on 5/29/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ServerConnectionHeaderView: View {
+    var title: String
+    var subtitle: String
+    var body: some View {
+        VStack(spacing: 20) {
```

Comment:
I think the space is not the same between the icon - title (24) and between the title - description (4) ?